common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1073,7 +1064,7 @@ def has_course(self, course_key, ignore_case=False, **kwargs):
                     course_query[key] = re.compile(r"(?i)^{}$".format(course_query[key]))
         else:
             course_query = {'_id': location.to_deprecated_son()}
-        course = self.collection.find_one(course_query, fields={'_id': True})
+        course = self.collection.find_one(course_query)
```

Comment:
I see these were forward-compatibility tweaks for pymongo3.0
Was this previously only returning the _id as a performance tweak?  Do we think fetching the full document will hurt us?